scheduling/src/main/java/io/helidon/scheduling/Scheduling.java

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.helidon.scheduling;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+import io.helidon.common.configurable.ScheduledThreadPoolSupplier;
+
+/**
+ * Scheduling periodically executed task with specified fixed rate or cron expression.
+ *
+ * <pre>{@code
+ * Scheduling.fixedRate()
+ *      .delay(2)
+ *      .task(inv -> System.out.println("Executed every 2 seconds"))
+ *      .build();
+ * }</pre>
+ *
+ * <pre>{@code
+ * Scheduling.cron()
+ *      .expression("0 45 9 ? * *")
+ *      .task(inv -> System.out.println("Executed every day at 9:45"))
+ *      .build()
+ * }</pre>
+ */
+public class Scheduling {
+
+    private Scheduling() {
+        //hidden constructor
+    }
+
+    /**
+     * Build a task executed periodically at a fixed rate.
+     *
+     * @return this builder
+     */
+    public static FixedRateBuilder fixedRate() {
```

Comment:
The dev guidelines expect the method to be called `builder`, so in case we have two builders,
we should have `fixedRateBuilder` and `cronBuilder` methods.

Reply:
fixed